pandas/core/groupby.py

```diff
@@ -1023,6 +1024,61 @@ def tail(self, n=5):
         tail = obj[in_tail]
         return tail
 
+    def pipe(self, func, *args, **kwargs):
+        """
+        Apply func(self, \*args, \*\*kwargs)
```

Comment:
would like an example in the docs using this groupby.pipe in a relevant/realistic situation (put same example in the whatsnew as well)